draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -156,6 +157,34 @@ For any remote host ICE candidate received by the ICE agent, the following proce
 Multicast DNS resolution might end up retrieving both an IPv4 and IPv6 address.
 In that case, the IPv6 address may be used preferably to the IPv4 address.
 
+### Handling of Peer-Reflexive Remote Candidate
+
+A peer-reflexive remote candidate could be learnt and constructed from the
+source transport address of the STUN Binding request as an ICE connectivity
+check. The peer-reflexive candidate could share the same address as a host
+remote candidate that will be signaled or has been signaled, received and is in
+the process of name resolution. Therefore, the IP obfuscation with mDNS requires
+special handling of peer-reflexive candidates as follows, in addition to the
+standard procedure of constructing a peer-reflexive remote candidate and the
+elimination procedure of redundant candidates defined in {{RFC8445}}.
```

Comment:
If we completely block the exposure of IP addresses of prflx candidates, I think it would suffice to handle the issue. It is not clear to me though if exposing prflx candidate IPs is not spec compliant.